rosie/src/main/java/com/karumi/rosie/view/RosieFragment.java

```diff
@@ -59,9 +59,7 @@
       Bundle savedInstanceState) {
     injectDependencies();
     int layoutId = getLayoutId();
-    View view = inflater.inflate(layoutId, container, false);
-    ButterKnife.bind(this, view);
-    return view;
+    return inflater.inflate(layoutId, container, false);
```

Comment:
I was told no to do that and initialise it in onCreateView of the Fragment, I really don't have any preference in any of those solutions. @pedrovgs ?